Samples/Mutation_Files/Mutation.py

```diff
@@ -0,0 +1,48 @@
+import random
+
+# Example data
+class_times = [
+    {"section": "A", "teacher": "sashi kumar", "time_slot": 1, "classRoom": "LT01"},
+    {"section": "B", "teacher": "subhankar goyal", "time_slot": 2, "classRoom": "LT01"},
+    {"section": "A", "teacher": "mehul manu", "time_slot": 3, "classRoom": "LT02"},
+    {"section": "C", "teacher": "kiran kher", "time_slot": 4, "classRoom": "LT02"},
+    {"section": "B", "teacher": "devender", "time_slot": 4, "classRoom": "LT02"}
+    # Add more class entries here
+]
+

```

Comment:
Add all the constraints classroom availability , teacher availability , subject class weightage ,teacher id, subject id  (can check the Fitness_function for all the contraints.